### PR TITLE
Update demo apps for testing new additional information in MSACErrorReport

### PR DESCRIPTION
**Pull request by AnastasiaKubova**
Things to consider before you submit the PR:

* [ ] ~Has `CHANGELOG.md` been updated?~
* [ ] ~Are tests passing locally?~
* [x] Are the files formatted correctly?
* [ ] ~Did you add unit tests?~
* [x] Did you check UI tests on the sample app? They are not executed on CI.
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Update demo apps for testing new additional information in `MSACErrorReport`.

## Related PRs or issues

[AB#88190](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/88190)

### CODE REVIEW

File: Sasquatch/SasquatchObjC/AppDelegate.m [hunk 1]
```diff
@@ -291,16 +291,16 @@ - (void)analytics:(nonnull MSACAnalytics *)analytics
 #pragma mark - MSACCrashesDelegate
 
 - (BOOL)crashes:(nonnull MSACCrashes *)crashes shouldProcessErrorReport:(nonnull MSACErrorReport *)errorReport {
-  NSLog(@"Should process error report with: %@", errorReport.exceptionReason);
+  NSLog(@"Should process error report with: %@", [errorReport description]);
```
Comment (DmitriyKirakosyan) [inline Sasquatch/SasquatchObjC/AppDelegate.m:294]: Any reason behind the change `.` to `[]`?

File: Sasquatch/SasquatchSwift/AppDelegate.swift [hunk 2]
```diff
@@ -212,18 +212,28 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CrashesDelegate, CLLocati
...
-    // return true if the crash report should be processed, otherwise false.
+    if errorReport.exceptionReason != nil {
+      NSLog("Should process error report with: %@", errorReport.description);
+    }
     return true
   }
 
   func crashes(_ crashes: Crashes!, willSend errorReport: ErrorReport!) {
+    if errorReport.exceptionReason != nil {
+      NSLog("Will send error report with: %@", errorReport.description);
```
Comment (DmitriyKirakosyan) [inline Sasquatch/SasquatchSwift/AppDelegate.swift:223]:
```suggestion
      NSLog("Will send error report: %@", errorReport.description);
```

File: Sasquatch/SasquatchSwift/AppDelegate.swift [hunk 3]
```diff
@@ -212,18 +212,28 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CrashesDelegate, CLLocati
   // Crashes Delegate
 
   func crashes(_ crashes: Crashes, shouldProcess errorReport: ErrorReport) -> Bool {
-
-    // return true if the crash report should be processed, otherwise false.
+    if errorReport.exceptionReason != nil {
+      NSLog("Should process error report with: %@", errorReport.description);
```
Comment (DmitriyKirakosyan) [inline Sasquatch/SasquatchSwift/AppDelegate.swift:216]:
```suggestion
      NSLog("Should process error report with description: %@", errorReport.description);
```

File: Sasquatch/SasquatchSwift/AppDelegate.swift [hunk 4]
```diff
@@ -212,18 +212,28 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CrashesDelegate, CLLocati
...
 
   func crashes(_ crashes: Crashes!, willSend errorReport: ErrorReport!) {
+    if errorReport.exceptionReason != nil {
+      NSLog("Will send error report with: %@", errorReport.description);
+    }
   }
 
   func crashes(_ crashes: Crashes!, didSucceedSending errorReport: ErrorReport!) {
+    if errorReport.exceptionReason != nil {
+      NSLog("Did succeed error report sending with: %@", errorReport.description);
```
Comment (DmitriyKirakosyan) [inline Sasquatch/SasquatchSwift/AppDelegate.swift:229]:
```suggestion
      NSLog("Did succeed sending error report: %@", errorReport.description);
```

File: Sasquatch/SasquatchSwift/AppDelegate.swift [hunk 5]
```diff
@@ -212,18 +212,28 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CrashesDelegate, CLLocati
...
+    }
   }
 
   func crashes(_ crashes: Crashes, didFailSending errorReport: ErrorReport, withError error: Error?) {
+    if errorReport.exceptionReason != nil {
+      NSLog("Did fail sending report with: %@, and error: %@", errorReport.description);
```
Comment (DmitriyKirakosyan) [inline Sasquatch/SasquatchSwift/AppDelegate.swift:235]:
```suggestion
      NSLog("Did fail sending error report: %@, with error: %@", errorReport.description, error);
```

File: Sasquatch/SasquatchSwift/AppDelegate.swift (hunk 2 above)
Comment (AnastasiaKubova) [reply Sasquatch/SasquatchSwift/AppDelegate.swift:223]: Please pay attention that all these logs were copied from other our test apps, let's keep it as it is.

**Review by DmitriyKirakosyan — APPROVED** [review]

**Pull request merged**